dietpi/dietpi-software

```diff
@@ -6800,28 +6801,40 @@ _EOF_
 			Download_Test_Media
 		fi
 
-		software_id=68 # Remot3.it
+		software_id=68 # Remote.It
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Installing
 
+			local licensekey=
+            G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+			G_WHIP_INPUTBOX 'If you do not already have the Remote.It Desktop application on your laptop, install it.
+https://www.remote.it/download\n
+Please enter your license key obtained from the Remote.It Desktop application.
+[Account] -> [Licensing]\n\n
+NB: This is optional, select "Skip" to leave it empty.
+    If so, register the device using the claim code shown during the installation process.\n
+    See below for instructions on how to register the device using a claim code:
+    https://docs.remote.it/software/device-package/installation#3.-claim-and-register-the-device\n' && licensekey=$G_WHIP_RETURNED_VALUE
+            G_EXEC export R3_REGISTRATION_CODE="$licensekey"
+            G_EXEC export R3_PLATFORM_CODE="1072"
```

Comment:
Is it possible to enter the license key manually afterwards? I would like to keep it available as automated install option, where neither interactive input is possible, nor is any output from the install (like the "claim code") assured to be reviewable afterwards.

We could/should also add an optional `dietpi.txt` setting for the license key.

Reply:
At this time, it is not possible to enter the license key manually after installation. It is used if it is set as an environment variable during installation, and if not, a claim code is shown at the end of the installation process. 
In that case, the claim code will be found in /etc/remoteit/config.json.
```
$ grep "claim" /etc/remoteit/config.json
    "claim": "JZQFXG4N"
```

Reply:
Okay, then I suggest to allow non-interactive installs, setting the registration code via `dietpi.txt` setting, and we will add the info to our [docs](https://dietpi.com/docs/software/remote_desktop/#remot3it) about how to retrieve the claim code from this file.

Reply:
Thank you for your suggestion. `dietpi.txt` in your first suggestion, but how do I use this? If you have a technical manual, it would be helpful to know its location.

And next suggestion, yes, if `/etc/remoteit/config.json` exists and remote.it is already configured, you can skip entering the registration code. Just an overwrite install will do.

Reply:
Do you mean to define `SOFTWARE_REMOTEIT_REG_CODE` (for example) in `dietpi.txt` with an empty value and set their code that the user has texted in?

Reply:
> Do you mean to define `SOFTWARE_REMOTEIT_REG_CODE` (for example) in `dietpi.txt` with an empty value and set their code that the user has texted in?

Yes exactly. Probably even have the setting commented by default.

Reply:
Since providing users with multiple choices installation way may lead to confusion, we decided to unify the installation method by claim code on DietPi. Also problems such as incorrect value of the license key can be expected.
Instead, we will add a process that shows the claim code to the user upon completion of the remote.it installation and show the link to our web manual.

Reply:
Sounds good. We'll document this in our online docs the same way, and how to review the claim code.